deploy/rbac.yaml

```diff
@@ -100,6 +100,7 @@ rules:
   resources:
   - issuers
   - certificates
+  - certificates/status
```

Comment:
please also update the bundle.yaml

Reply:
We should not have it at all. It is needed only for test.